tests/network/bandwidth/test_bandwidth.py

```
@@ -0,0 +1,32 @@
+    BANDWIDTH_SECONDARY_IFACE_NAME,
+    active_tcp_connection_output,
+)
+
+_BANDWIDTH_TOLERANCE: Final[float] = 1.1
+
+
+def test_bandwidth_limit_enforced_on_secondary_interface(
+    subtests,
+    server_vm,
+    client_vm,
+    bandwidth_nad,
```

Ruff (0.15.7)

[warning] Unused function argument: `bandwidth_nad`

(ARG001)